<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditThermalEffectsDialog</class>
 <widget class="QDialog" name="EditThermalEffectsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Thermal Effects</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="selectedThermalEffectsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Selected Thermal Effects</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QTextEdit" name="selectedThermalEffectsTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="barLabel">
     <property name="text">
      <string>Bar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="ComboBox" name="barComboBox"/>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QGroupBox" name="temperatureChangeGroupBox">
     <property name="title">
      <string>Temperature Change</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="riseRadioButton">
        <property name="text">
         <string>Rise</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="fallRadioButton">
        <property name="text">
         <string>Fall</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="temperatureChangeLabel">
        <property name="text">
         <string>dt</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="temperatureChangeDoubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="thermalCoefficientLabel">
     <property name="text">
      <string>a</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QDoubleSpinBox" name="thermalCoefficientDoubleSpinBox">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="editThermalEffectsPushButton">
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="icon">
      <iconset resource="../../rsc.qrc">
       <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
